foundation/foundation-mda/src/main/resources/templates/deployment/argocd/v2/configuration-store.yaml.vm

```diff
@@ -1,14 +1,16 @@
 apiVersion: argoproj.io/v1alpha1
 kind: Application
 metadata:
+  annotations:
+    argocd.argoproj.io/sync-wave: "-2"
```

Comment:
I: assign lower number sync wave to ensure the configuration store app starts up before all other apps